db/CSV_Writers/MISC/userTableGenerator.js

```diff
@@ -0,0 +1,17 @@
+const path = require('path');
+const fs = require('fs');
+const Sequelize = require('sequelize');
+
+// I'm not creating tables anymore because I don't need them, still keeping this just in case
```

Comment:
if youre not using this, might be good to delete. if you made it work before, you should be able to do it again. might make the code cleaner/easier to follow.

Reply:
I do agree but I want to hold on for now because I had to make some adjustments to the sequelize configuration that basically made the models into functions instead of models. So this is just a reminder to change it back otherwise. I also have a feeling that I will need to revisit the adding table logic in deployment. But maybe I can just make the batches smaller. Until then keeping it in the MISC folder.